vaadinfrontend/frontend/styles/views/project/project-view.css

```diff
@@ -1,3 +1,4 @@
+/* Subdivides the project-view-page into 4 columns and rows with equal spacing on which the components can be allocated*/
```

Comment:
wouldn't a html `id` fit better than a HTML class in this case?

Reply:
That's a valid point i adjusted the css accordingly